packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -257,7 +258,7 @@ export default defineComponent({
 
     const capabilityFilesTags = useCapabilityFilesTags()
     const hasTags = computed(() => {
-      return props.tagsEnabled && unref(capabilityFilesTags)
+      return props.tagsEnabled && unref(capabilityFilesTags) && unref(isUserContext)
```

Comment:
We need do check if userContext otherwise, showing tags will result in a 401 error in case user is on a public link and not authenticated

Reply:
Not something for now necessarily, but IMO we should move `unref(capabilityFilesTags) && unref(isUserContext)` to the panel and only use the `tagsEnabled` property here.

Reply:
Unfortunately not possible without refactoring, as FileDetails will also be called on a single shared resource and doesn't have the tagsEnabled prop accessible (defaults to true)